src/query/settings/src/settings_default.rs

```diff
@@ -357,7 +357,7 @@ impl DefaultSettings {
                     display_in_show_settings: true,
                 }),
                 ("use_parquet2", DefaultSettingValue {
-                    value: UserSettingValue::UInt64(1),
+                    value: UserSettingValue::UInt64(0),
```

Comment:
Why we disable the `use_parquet2` by default here? @RinChanNOWWW

Reply:
When `parquet_rs` is finely tested, we will remove `parquet2`, so we use `parquet_rs` by default now.